style.scss

```diff
@@ -11,144 +12,270 @@ body {
+        height: 400px;
+
+        @media (min-width: $screen-md) {
+            height: 600px;
+        }
+    }
+
+    .text-about-us {
+        text-align: center;
+        padding-top: 100px;
+        margin: 0 300px 0 10px;
+        font-size: 1rem;
+
+        @media (min-width: $screen-sm) {
+            margin: 0 500px 0 30px;
+            font-size: 1.5rem;
+        }
+
+        @media (min-width: $screen-md) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+
+        @media (min-width: $screen-lg) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+    }
+
+    .main-title {
+        color: $primary;
+        text-align: center;
+        text-wrap: balance;
+        margin-bottom: 1rem;
+        font-size: 1rem;
+
+        @media (min-width: $screen-xs) {
+            font-size: 2rem;
+        }
+    }
+
+    .cards-container {
+        display: flex;
+        flex-direction: column;
+
+        @media (min-width: $screen-xs) {
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    .card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 1rem;
+        gap: 1rem;
+        background-color: $light;
+        color: $black;
+
+        @media (min-width: $screen-xs) {
+            width: 50%;
+        }
+
+        @media (min-width: $screen-md) {
+            width: 33.3333333%;
+        }
+
+        &:nth-of-type(odd) {
+            background-color: $secondary-light;
+            color: $light;
+
+            @media (min-width: $screen-xs) {
+                background-color: $light;
+                color: $black;
+            }
+
+            @media (min-width: $screen-md) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        &:nth-of-type(1),
+        &:nth-of-type(4),
+        &:nth-of-type(5) {
+            @media (min-width: $screen-xs) and (max-width: ($screen-md - 1)) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        .icon {
+            width: 50px;
+            height: 50px;
+        }
+
+        .title {
+            text-align: center;
+            font-weight: 400;
+            margin: 0;
+            font-size: 0.8rem;
+
+            @media (min-width: $screen-xs) {
+                font-size: 1rem;
+            }
+        }
+    }
 }
 
-.footer {
-    background-color: $primary-lighter;
+footer {
+    background-color: $primary-light;
     display: flex;
-    flex-direction: row;
     justify-content: center;
-}
 
-.footer-container {
-    display: flex;
-    flex-direction: column;
-    text-align: center;
-}
+    .footer-container {
+        display: flex;
+        flex-direction: column;
+        text-align: center;
+    }
 
-.footer-logo {
-    width: 200px;
-}
+    .footer-logo {
+        width: 50px;
+        height: 100px;
```

Comment:
Please use the same approach as I used for the same image in `header`: with no `height`, and `max-width` instead of width.